examples/pytorch/test_accelerate_examples.py

```diff
@@ -93,7 +93,7 @@ def test_run_glue_no_trainer(self):
             --with_tracking
         """.split()
 
-        if is_cuda_and_apex_available():
+        if is_torch_fp16_available_on_device(torch_device):
             testargs.append("--fp16")
```

Comment:
I don't know the reason why we test with `fp16` only if  `apex` is available. The change here will means we will test it as long as CI is running on GPU. Not sure if this is what we expect.

@muellerzr Do you happen to know the story here (about the `apex` part)?

Reply:
I think it was a bad copy/paste, because I wouldn't have had that in there on purpose.

Reply:
OK thank you for the feedback ❤️